test-projects/login/index.js

```diff
@@ -36,6 +45,23 @@ keystone.createList('Post', {
   },
 });
 
+keystone.createList('ListWithPlugin', {
```

Comment:
Why are we adding this to the login test? And I'm not sure we need a cypress test. It should probably be part of API testing and ideally its own section?

I actually wouldn't mind if this (and other plugins) are published as their own npm modules with self contained tests and everything... does that mean outside mono repo @jesstelford, @timleslie let's discuss.

Reply:
I don't mind having one of our cypress tests utilise these plugins purely from a smoke testing point of view. I would put it onto the `basic` project rather than the `login` project, as that's where we have the kitchen sink.

The plugins should definitely have their own unit tests, but there's no reason for them to live outside the mono repo. The `@keystone-alpha/list-plugin` package will contains all of our sanctioned plugins, which we should be testing as much as possible.

Reply:
I'm fine with all that :)

Reply:
initially used `basic` project but soon realised that due to no authentication the `byTracking` features would not work.